templates/repo/commit_statuses.tmpl

```diff
@@ -4,10 +4,11 @@
 		{{range .Statuses}}
 			<div class="ui item singular-status">
 				<span>{{template "repo/commit_status" .}}</span>
-				<span class="ui">{{.Context}} <span class="text grey">{{.Description}}</span></span>
+				<span class="ui">{{.Context}}</span>
 				{{if .TargetURL}}
 					<div class="ui right"><a href="{{.TargetURL}}" target="_blank" rel="noopener noreferrer">{{$.root.i18n.Tr "repo.pulls.status_checks_details"}}</a></div>
 				{{end}}
+				<div class="ui markup mt-4">{{.Description|Md2html}}</div>
```

Comment:
Can you share some screenshots about this?

IIRC the `details` links should appear at the bottom (just like GitHub does). This PR changes the position.

Reply:
I tried to keep the `details` links where they were, based on what gitea currently does

![qq](https://user-images.githubusercontent.com/19575594/146137214-36ee67bc-ff7f-40af-bad0-420533e1e986.png)
![qq2](https://user-images.githubusercontent.com/19575594/146137209-30774236-3c7b-4582-a2e3-9e329ac3e868.png)

Reply:
Do i change these or we keep it this way?

Reply:
Oh, that's fine to me 😊